certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -341,23 +283,26 @@ public void generateCert(File csrFile, File caKey, File caCert, File crtFile, Su
                 log.warn("{} cannot be deleted", sna.getName());
             }
         }
-        if (openSslConf != null) {
-            if (!openSslConf.delete()) {
-                log.warn("{} cannot be deleted", openSslConf.getName());
-            }
-        }
 
         // We need to remove CA serial file
         Files.deleteIfExists(Paths.get(caCert.getPath().replace(".crt", ".srl")));
     }
 
+    private Path createDefaultConfig() throws IOException {
+        try (InputStream is = getClass().getClassLoader().getResourceAsStream("openssl.conf")) {
+            Path openSslConf = Files.createTempFile("openssl-", ".conf");
+            Files.copy(is, openSslConf, StandardCopyOption.REPLACE_EXISTING);
+            return openSslConf;
```

Comment:
again here, I see that where we use `createDefaultConfig`, we need a `File` to we call `toFile` on the returned `Path`. Why not return the `File` instance directly?

Reply:
Same reason really, and I found using Paths everywhere was just easier.